src/views/activity/Activity.vue

```diff
@@ -4,8 +4,6 @@ div
 
   div.mb-2
     ul.list-group.list-group-horizontal-md.mb-3(style="font-size: 0.9em; opacity: 0.7")
-      li.list-group-item.pl-0.pr-3.py-0(style="border: 0")
-        | #[b Host:] {{ host }}
```

Comment:
This should be kept as the view host (where the data is from) may not be the same as the server hostname (where the data is stored). 

Not easy to know if you haven't imported buckets from several hosts... :)

Reply:
Got it, I've added this back in and made a couple of other tweaks.